P/Prjtrellis/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Prjtrellis"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/YosysHQ/prjtrellis.git", "dff1cbcb1bd30de7e96f8a059f2e19be1bb2e44d")
+]
+
+dependencies = [
+    Dependency("Python_jll"),
+    Dependency("boost_jll"; compat="=1.76.0"), # max gcc7
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd prjtrellis/libtrellis
+cmake -DCMAKE_INSTALL_PREFIX=${prefix}
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(p -> Sys.islinux(p), supported_platforms())
+platforms = expand_cxxstring_abis(platforms)
+# For some reason, building for CXX03 string ABI doesn't actually work, skip it
+filter!(x -> cxxstring_abi(x) != "cxx03", platforms)
```

Comment:
Do you actually see this?

Reply:
Yes, I think it is related to Boost (again).